policy_packs/aws/ssm/enforce_install_packages_on_instances_using_ssm_package_installer/README.md

```diff
@@ -0,0 +1,163 @@
+---
+categories: ["security"]
+primary_category: "security"
+---
+
+# Enforce Install Packages On Instances Using AWS SSM Package Installer
```

Comment:
```suggestion
# Enforce Install Packages on AWS EC2 Instances Using AWS SSM Package Installer
```

I don't think `on` is capitalized in sentence case. Also, do we have any other titles where we have 2 or more services? Do we always include the cloud provider name, or just for the first service?

Reply:
The dir name and file names should be updated too

Reply:
Updated.

Reply:
@SumitPopat Do we need `AWS` twice? I'm still not sure on that part. Do we have any other examples of multiple services in a title?